src/components/accounts/CreateAccount.js

```diff
@@ -290,15 +296,21 @@ const mapDispatchToProps = {
     redirectTo
 };
 
-const mapStateToProps = ({ account, status }, { match }) => ({
-    ...account,
-    localAlert: status.localAlert,
-    mainLoader: status.mainLoader,
-    isMobile: status.isMobile,
-    fundingContract: match.params.fundingContract,
-    fundingKey: match.params.fundingKey,
-    fundingAccountId: match.params.fundingAccountId,
-});
+const mapStateToProps = (state, ownProps) => {
+    const { account, status } = state;
+    const { match } = ownProps;
+
+    return {
+        ...account,
+        localAlert: status.localAlert,
+        mainLoader: status.mainLoader,
+        isMobile: status.isMobile,
+        fundingContract: match.params.fundingContract,
+        fundingKey: match.params.fundingKey,
+        fundingAccountId: match.params.fundingAccountId,
+        nearTokenFiatValueUSD: selectNearTokenFiatValueUSD(state)
```

Comment:
Hey look! A SELECTOR!!! ❇️ 💖 🤯  :D I think this might be the _only_ selector in a `mapStateToProps()` in the entire codebase right now :D. It feels lonely.